Lib/pathlib/_abc.py

```diff
@@ -206,10 +206,14 @@ class PurePathBase:
     )
     pathmod = os.path
 
-    def __init__(self, *paths):
-        self._raw_paths = paths
+    def __init__(self, *args):
+        self._raw_paths = self._load_args(args)
         self._resolving = False
 
+    def _load_args(self, args):
```

Comment:
Add a docstring (which all functions should have ;-) explaining that subclasses can override and avoid adding a subclass .__init__ that calls super with packing and uppacking and a noticeable slowdown.

Reply:
I copied your wording as it's spot on :)